Makefile

```diff
@@ -80,6 +80,7 @@ gen-mocks: ## Generate mocks
 .PHONY: gen-docs
 gen-docs: ## Generate docs for commands
 	@echo "==> Generating docs"
+	rm -rf ./docs/command
```

Comment:
flyby: cater for deleting/renaming commands in our docs generation

Reply:
https://github.com/mongodb/mongocli/blob/2ee57842ac8088e08c451a50506a304641ca17e9/internal/docs/main.go#L34-L36